src/main/webapp/WEB-INF/views/admin/edit-room.jsp

```diff
@@ -100,6 +98,52 @@
         </tr>
     </table>
 </div>
+<div id="deactivateModal" class="modal fade">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-body text-center">
+                <br>
+                <h2><spring:message code="room.deactivate"/></h2>
+                <p class="cancelName ">
+                    <spring:message code="booking.confirmCancelQuestion2"/>
+                </p>
+                <button id="deactivateYesButton" class="btn  btn-success">
+                    <spring:message code="room.yes_button" />
+                </button>
+                <button id="deactivateNoButton" class="btn btn-danger" data-dismiss="modal">
+                    <spring:message code="room.no_button"/>
+                </button>
+            </div>
+        </div>
+    </div>
+</div>
+<div id="activateModal" class="modal fade">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-body text-center">
+                <br>
+                <h2><spring:message code="room.activate"/></h2>
+                <button id="activateYesButton" class="btn  btn-success">
+                    <spring:message code="room.ok_button" />
+                </button>
+            </div>
+        </div>
+    </div>
+</div>
 
+<script src="http://cdn.jsdelivr.net/jquery.validation/1.15.0/jquery.validate.min.js"></script>
+<script src="http://cdn.jsdelivr.net/jquery.validation/1.15.0/additional-methods.min.js"></script>
 <script src='resources/js/adminRoomSearch.js'></script>
-</body>
\ No newline at end of file
+<script src='resources/js/admin-deactivate-room-button.js'></script>
+
+<c:choose>
+    <c:when test="${pageContext.response.locale=='ua'}">
+        <script src="http://ajax.aspnetcdn.com/ajax/jquery.validate/1.15.0/localization/messages_uk.js"></script>
+        <script src="resources/js/lib/messages-ua.js"></script>
+    </c:when>
+    <c:when test="${pageContext.response.locale!= 'ua'}">
```

Comment:
You needn't do that. Similar was done in template.jsp. If somebody is added third language there was a conflict.